cluster-autoscaler/cloudprovider/aws/aws_cloud_provider.go

```diff
@@ -343,12 +346,36 @@ func BuildAWS(opts config.AutoscalingOptions, do cloudprovider.NodeGroupDiscover
 		defer config.Close()
 	}
 
+	// Generate EC2 list
+	var instanceTypes map[string]*InstanceType
+	if opts.StaticInstanceList {
+		klog.Warning("Use static EC2 Instance Types, list could be outdated")
+		instanceTypes = GetStaticEC2InstanceTypes()
```

Comment:
Possibly this method can also return the date when the list was last updated. Then you could add it to warning message to give clear singal how up-to-date data is.

Reply:
This is great suggestion.
I create a const and add constructions in the doc that everytime the list is updated, people need to update date correspondingly.

Reply:
W1014 17:25:22.612552   75153 aws_cloud_provider.go:354] Use static EC2 Instance Types and list could be outdated. Last update time: 2019-10-14
I1014 17:25:22.612701   75153 reflector.go:120] Starting reflector *v1.PersistentVolumeClaim (0s) from k8s.io/client-go/informers/factory.go:134